arviz/data/inference_data.py

```diff
@@ -76,34 +87,53 @@ def __init__(self, **kwargs):
 
         """
         self._groups = []
-        key_list = [key for key in SUPPORTED_GROUPS if key in kwargs]
+        self._groups_warmup = []
+        save_warmup = kwargs.pop("save_warmup", False)
+        key_list = [key for key in SUPPORTED_GROUPS + SUPPORTED_GROUPS_WARMUP if key in kwargs]
         for key in kwargs:
-            if key not in SUPPORTED_GROUPS:
+            if key not in SUPPORTED_GROUPS + SUPPORTED_GROUPS_WARMUP:
```

Comment:
`SUPPORTED_GROUPS + SUPPORTED_GROUPS_WARMUP1 should probably be made  a global so it doesn't need to be repeated everywhere